docs/quick_start.rst

```diff
@@ -21,3 +21,38 @@ Output:
  * output.nwb: NWB file including spike times
  * log.txt: run log
  * qc_figs: index.html includes cell figures and feature table and sweep.html includes sweep figures
+
+
+Step-by-step guideline
+----------------------
+
+ * create a clean environment with conda
+
+.. code-block:: bash
+
+    conda create -n myenv python=3.6
+    conda activate myenv
+
+ * install ipfx from pypi with "pip install ipfx"
+
+ * download example datasets (mouse data ~114 GB) with dandi
+
+.. code-block:: bash
+
+    pip install dandi
+    dandi download https://dandiarchive.org/dandiset/000020
+
+ * pick one dataset, e.g. sub-599387254_ses-601506492_icephys.nwb from mouse data, create output directory for saving the results, for example
+
+.. code-block:: bash
+
+     mkdir outputs
+     mkdir outputs/qc
```

Comment:
Also, I would like us to remove --qc_fig_dir parameter in favor of --save_figures that would create a relative directory automatically if save_figures=True. I created an issue https://github.com/AllenInstitute/ipfx/issues/454

Reply:
removed it as well, thanks.